blocks/header/header.js

```diff
@@ -355,13 +355,70 @@ function settingAltTextForSearchIcon() {
   searchImage.setAttribute('title', listOfAllPlaceholdersData.searchAltText);
 }
 
-async function fetchingPlaceholdersData() {
+async function fetchingPlaceholdersData(placeholdersData) {
   listOfAllPlaceholdersData = await fetchLanguagePlaceholders();
   const hamburger = document.querySelector('.nav-hamburger');
-  hamburger.setAttribute('title', listOfAllPlaceholdersData.hamburgerAltText);
+  hamburger.setAttribute('title', placeholdersData.hamburgerAltText);
   makeImageClickableNSettingAltText();
   settingAltTextForSearchIcon();
 }
+const toggleExpandLanguageSelector = (e) => {
+  const toggleContainer = e.currentTarget;
+  if (toggleContainer) {
+    const toggler = toggleContainer.querySelector('div.language-toggle');
+    const toggleContent = toggleContainer.querySelector('div.language-content');
+    if (e.type === 'click' && toggleContent) {
+      if (toggleContainer.classList.contains('nav-item-expanded-active')) {
+        toggleContainer.classList.remove('nav-item-expanded-active');
+        toggler.setAttribute('aria-expanded', false);
+        toggleContent.classList.remove('nav-item-content-expanded');
+      } else {
+        toggleContainer.classList.add('nav-item-expanded-active');
+        toggler.setAttribute('aria-expanded', true);
+        toggleContent.classList.add('nav-item-content-expanded');
```

Comment:
You can try `toggle` instead of `add` and `remove` 

```js
const isExpanded = toggleContainer.classList.contains('nav-item-expanded-active');    

if (e.type === 'click') {
    toggleContainer.classList.toggle('nav-item-expanded-active');
    toggler.setAttribute('aria-expanded', !isExpanded);
    toggleContent.classList.toggle('nav-item-content-expanded');
}
```

Reply:
Done